routes/cart.js

```diff
@@ -8,9 +8,33 @@ const CartServiceInstance = new CartService();
 module.exports = (app) => {
     app.use('carts', router);
 
-    router.get('/mine', async(req, res, next) => {
+    /**
+     * @openapi
+     * /mine:
+     *   get:
+     *     tags:
+     *       - Cart
+     *     summary: Retrieve the user's cart
+     *     description: Loads the cart for the currently authenticated user
+     *     security:
+     *       - BearerAuth: []
+     *     responses:
+     *       200:
+     *         description: Successfully retrieved the user's cart
+     *         content:
+     *           application/json:
+     *             schema:
+     *               $ref: '#/components/schemas/Cart'
+     *       400:
+     *         description: Bad request
+     *       401:
+     *         description: Unauthorized
+     *       500:
+     *         description: Internal server error
+     */
+    router.get('/mine', async (req, res, next) => {
```

Comment:
ovdje se nalazi nas "open api dokument"

inace to bude u jedom fajlu (tako si ti to bija radija, cini mi se da si mi pokaziva)

mi smo koristili swagger-jsdoc paket, koji dozvoljava da za svaku rutu pojedinacno to radimo u vidu komentara iznad nje

napisemo komentar iznad svake rute kao ovdje gore i on nam zna kreirat dokument koji kasnije prosljedimo swagger-ui-express paketu